src/cbc_sdk/platform/devices.py

```diff
@@ -298,14 +317,19 @@ def nsx_remediation(self, tag, set_tag=True):
 
 
 class DeviceFacet(UnrefreshableModel):
-    """Represents a device field in a facet search."""
+    """
+    Represents a device field in a facet search.
+
+    *Faceting* is a search technique that categorizes search results according to common attributes. This allows
```

Comment:
We did an example on the Device we should add one for DeviceFacet

Reply:
I do not you'll have to try it and create one

Reply:
I have one, which is different from the one on the `device_query()` method.